sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/DateExpressionsSuite.scala

```diff
@@ -494,7 +494,7 @@ class DateExpressionsSuite extends SparkFunSuite with ExpressionEvalHelper {
     // Valid range of DateType is [0001-01-01, 9999-12-31]
     val maxMonthInterval = 10000 * 12
     checkEvaluation(
-      AddMonths(Literal(Date.valueOf("0001-01-01")), Literal(maxMonthInterval)), 2933261)
+      AddMonths(Literal(LocalDate.parse("0001-01-01")), Literal(maxMonthInterval)), 2933263)
```

Comment:
I had to replace `Date.valueOf` by `LocalDate.parse` due to migration on another calendar from Julian to Gregorian. The expected number of days is easy to get:
```scala
val a = LocalDate.of(10001, 1, 1).toEpochDay
println(a)
2933263
```

Reply:
can we place the harcoded `2933263` with `LocalDate.of(10001, 1, 1).toEpochDay`?